### PR TITLE
Add node migration guide

### PR DESCRIPTION
**Pull request by jianoaix**

### CODE REVIEW

File: docs/eigenda/operator-guides/upgrades/system-upgrades.md [hunk 1]
```diff
@@ -6,20 +6,27 @@ sidebar_position: 2
...
+
+**Old machine**:
+1. Opt-out of all the quorums
+2. Keep the node running for >1h
+3. Continue to keep the node running while spinning up the new machine
+
+**New machine**:
+1. Copy over the files from old machine located at `NODE_DB_PATH`
+2. Start the EigenDA Node with the files copied from the old machine (e.g. `docker-compose up -d`) and make sure node is reachable
+3. Opt-in the quorums
```
Comment (mooselumph) [inline docs/eigenda/operator-guides/upgrades/system-upgrades.md:30]: Should we mention IP addresses? I think this flow is assuming that the two instances of the node have different sockets and thus are not trying to use the same IP address port. Should we clarify this assumption?

File: docs/eigenda/operator-guides/upgrades/system-upgrades.md [hunk 2]
```diff
@@ -6,20 +6,27 @@ sidebar_position: 2
...
-Generally speaking, you should follow the following sequence in order to maintain data integrity: 
-1. Provision the new storage location. Suppose that the variable `$NODE_DB_PATH_HOST_NEW` points to your new storage location.
-2. Stop the DA node (e.g. `docker-compose down`)
-3. Copy the the contents of the old storage location to the new storage location (e.g. `cp -r $NODE_DB_PATH_HOST $NODE_DB_PATH_HOST_NEW`)
-4. Update the DA node configuration (e.g. set the `NODE_DB_PATH_HOST` variable in your `.env` file to the value of `$NODE_DB_PATH_HOST_NEW`).
-5. Restart the DA node (e.g. `docker-compose up -d`)
\ No newline at end of file
+Generally speaking, suppose you want to migrate node to a new machine, you should follow the following sequence in order to maintain data integrity and meet the protocol SLA:
+
+**Old machine**:
```
Comment (mooselumph) [inline docs/eigenda/operator-guides/upgrades/system-upgrades.md:22]: Should we mention backing up keys and other config data as well?

Comment (jianoaix) [reply docs/eigenda/operator-guides/upgrades/system-upgrades.md:22]: Yeah good point

File: docs/eigenda/operator-guides/upgrades/system-upgrades.md (hunk 1 above)
Comment (jianoaix) [reply docs/eigenda/operator-guides/upgrades/system-upgrades.md:30]: Yea good point again. To make this migration without downtime/violating SLA, this assumes a new IP. We should make it more clear here.

File: docs/eigenda/operator-guides/upgrades/system-upgrades.md [hunk 3]
```diff
@@ -6,20 +6,26 @@ sidebar_position: 2
...
+1. Backup the keys stored at the machine as well as other configs you want to migrate
+2. Opt-out of all the quorums
+3. Keep the node running for >1h (note after this the node will stop receiving dispersal requests)
+4. Continue to keep the node running (for retrieval traffic) while spinning up the new machine
+
+**New machine**:
+1. Copy over the files from old machine located at `NODE_DB_PATH_HOST`
+2. Start the EigenDA Node (e.g. `docker-compose up -d`) with the files copied from the old machine (the file should be placed under the path per `NODE_DB_PATH_HOST`) and make sure node is reachable
+3. Opt-in the quorums with a new IP address (so the old machine remains reachable with original IP while this is setting up)
```
Comment (shrimalmadhur) [inline docs/eigenda/operator-guides/upgrades/system-upgrades.md:29]: `with a new IP address`
If someone has DNS configured - they would just need to repoint DNS after the switch right? should we put that instructions too?

Comment (jianoaix) [reply docs/eigenda/operator-guides/upgrades/system-upgrades.md:29]: Yes, just pointing the DNS to the new IP.

**Review by shrimalmadhur — APPROVED** [review]

**Review by mooselumph — APPROVED** [review]

**Pull request merged**